script/run_ck_profiler_gemm_with_csv_shapes.py

```diff
@@ -278,13 +278,13 @@ def main():
     shapes = tuples(filename)
 
     all_results = []
-    from tqdm import tqdm
     from functools import partial
     from os import path
 
     profiler_bin = path.join(args["build_dir"], "bin", "ckProfiler")
 
-    for s in tqdm(shapes):
+    total = len(shapes)
+    for idx, s in enumerate(shapes, 1): 
```

Comment:
for the manual script invocation I found the progress bar useful, since a single run can take 15-30 minutes, you want some visual confirmation that it's doing work and hasn't frozen. `pip install tqdm` is all you need

Reply:
This is a helper script and we don't invoke it from ROCm or CK internals, so it shouldn't matter if ROCm has the package. We also have other script in that folder which uses pandas which is not available by default. If we want to be really safe, we can try something like: 
```
try:
    from tqdm import tqdm as iterate
except ImportError:
    iterate = enumerate

for s in iterate(shapes):
...
```